qa/L0_backend_python/python_test.py

```diff
@@ -257,6 +257,30 @@ def test_string(self):
                 self.assertTrue(output0 is not None)
                 self.assertTrue(output0[0] == input_data.astype(np.bytes_))
 
+    def test_contiguous(self):
```

Comment:
This tests appears to be testing support for non contiguous output tensors. Thus test_non_contiguous would be more appropriate.